PULL_REQUEST_TEMPLATE/public_repository_request.md

```diff
@@ -0,0 +1,69 @@
+
+Use the checklist below to track your progress on these items. Add to the checklist if you complete additional steps
+
+- [ ] Primary repository branch is named "main"
+- [ ] Branch "main" protected
+- [ ] Only specifically noted users (tag users using @) have push privileges to repo on any branch
+- [ ] Pull request required before merging to "main"
+- [ ] At least one approval needed from maintainers (tag maintainers using @) before pull-requst merge
+- [ ] Conversations and change requests must be resolved before pull-request merge
+- [ ] Comprehensive README included [README.MD](README.MD)
+- [ ] Repository content reviewed for typos, grammar, and other inconsistencies
+- [ ] Repository git history is clean: no sensitive data commited and later removed from repository
+- [ ] *Contributing or style guide included [CONTRIBUTING.MD](CONTRIBUTING.MD)
+- [ ] *Pull request template included [PULL_REQUEST_TEMPLATE.MD](.github/PULL_REQUEST_TEMPLATE.MD)
+- [ ] *Code of conduct included [CODE_OF_CONDUCT.MD](CODE_OF_CONDUCT.MD)
+- [ ] *Automated testing (describe)
+
+*optional 
+
+**Describe expected level of outside interaction (popularity, issues, pull-requests)**
+
+## Repository requirements
+
+1.	The repository must be clean
+	- No absolute paths; all paths must be relative to project root directory
+    - No API keys, passwords, credentials, or other “secrets” in plain text. 
+    - Scan for secrets with [Talisman](https://thoughtworks.github.io/talisman/) and remove secrets
+    - If secrets exist in the git history, consider using a tool like [BFG Repo-Cleaner](https://rtyley.github.io/bfg-repo-cleaner/) or create a new repository and copy/paste all materials (excluding .git/)
+1.	The code must meet professional standards
+    - All commit messages, code comments and other documentation are professional and informative.
+    - A .gitignore file is required, and should at a minimum contain:
+      - .*
+      - !.gitignore
+    - All code should be documented, at a minimum describing what the code does, and a description of parameters, returned values and side-effects.
+1.	The code must have an approved OSS license, e.g., MIT, Apache-2.0, BSD-3, with the Metropolitan Council as the license holder.
+	- Data or documentation repositories should have an appropriate data license, e.g., PDDL-10, CC0-1.0, CC-BY-4.0.
+    - Public domain dedications (CC0, PDDL) relinquish all rights and are usual what we intend when we publish data, but you may find it appropriate in some conditions to maintain copyright and requesting minimal attribution.
+    - License should have a [Blue Oak Council](https://blueoakcouncil.org/list) rating of Gold, Silver, or Bronze.
+    - If a license other than MIT or Apache 2.0 is used, describe why in the request. 
+1.	A README.md file describes the purpose of the project
+1.	Add appropriate branch protection to the main branch
+
+## Manager approval
```

Comment:
I might rename this section, "Steps to approval", and change the wording to something like,

- [ ] If organizational risk is **not** considered low, forward this request as an email to IS security via the ServiceDesk.
- [ ] Your manager must approve of this pull-request. If they are on GitHub, request a review from them. Otherwise they can approve by sending an email to an admin of the repo, who can add the contents of the email to the pull-request and approve on their behalf.
- [ ] Once approved, request an admin to make the the repo public.